### ISSUE TITLE
Request to add an option to "addColumn" to disable as default

### ISSUE THREAD

**Issue by wattsie** [protonemedia/inertiajs-tables-laravel-query-builder#14]
Hi,
Wondering if it would be possible to add the option to add columns, but add them disabled by default.

Would be a bonus if the state can be saved, so that when returning to the table, that the selected columns visibility is maintained. 

Might be nice to extend to saving sort order etc.

```
    /**
     * Add a column to the query builder.
     *
     * @param string $key
     * @param string $label
     * @param bool $enabled
     * @return self
     */
    public function addColumn(string $key, string $label, bool enabled = true): self
    {
        $this->columns->put($key, [
            'key'     => $key,
            'label'   => $label,
            'enabled' => $enabled,
        ]);

        return $this;
    }
```

**Comment by neilgilmour** [comment]
I opened a PR for this a while ago, and we ended up making a pretty customised implementation, so I closed the PR. Having said that, we're still hoping to PR a solution to this if we can make it more generic...

**Comment by pascalbaljet** [comment]
Fixed with #30

**Comment by renepardon** [comment]
I try to use this to hide ID column (which is UUID). The v-show="showColumn('id')" works and the id columns is not displayed as expected, BUT I'm not enable to show the id column by checking the switch. Is this just a naming issue or expected behaviour?

My intention was to hide the column from being displayed but to be able to show it if explicitly requested.
